ide-alternatives.md

```diff
@@ -0,0 +1,21 @@
+# IDE and Swift-aware editor alternatives
+
+There are a few different options for doing the actual coding with completions etc depending on what platform you are using, a non-exhastive list:
+
+[Atom IDE support](https://atom.io/packages/ide-swift)
+
+[CLion](https://www.jetbrains.com/help/clion/swift.html)
+
+[Emacs plugin](https://github.com/swift-emacs/swift-mode)
+
+[VIM plugin](https://github.com/keith/swift.vim)
+
+[Visual Studio Code](https://code.visualstudio.com)
+
+[Visual Studio Code optional code completion plugin](https://marketplace.visualstudio.com/items?itemName=vknabel.vscode-swift-development-environment)
+
+[Xcode](https://developer.apple.com/xcode/ide/)
```

Comment:
one more idea, maybe add a line here that says something like:

`* Know about another IDE or IDE plugin? Please submit a PR to add them here!`

Reply:
Sure, why not,  added one at the end!